draft-irtf-cfrg-aegis-aead.md

```diff
@@ -226,6 +226,26 @@ informative:
         org: ANSSI
     date: 2014
 
+  SSI24:
+    title: "Bit-Wise Analysis for Forgery Attacks on AES-Based AEAD Schemes"
+    rc: "Advances in Information and Computer Security. IWSEC 2024. Lecture Notes in Computer Science, vol 14977"
+    seriesinfo:
+      DOI: 10.1007/978-981-97-7737-2_1
+    author:
+      -
+        ins: T. Shiraya
+        name: Takuro Shiraya
+        org: University of Hyogo
+      -
+        ins: K. Sakamoto
+        name: Kosei Sakamoto
+        org: University of Hyogo
+      -
+        ins: T. Isobe
+        name: Takanori Isobe
+        org: University of Hyogo; National Institute of Information and Communications Technology
```

Comment:
`National Institute of Information and Communications Technology` isn't mentioned in this paper so not sure if he's still there.